### ISSUE TITLE
Collection.update does not work on the server

### ISSUE THREAD

**Issue by varunwyest** [TheRealNate/meteor-react-native#92]
**Describe the bug**
When we use the Collection.update method it is taking a string as the ID which updates it perfectly on the local but is not able to update the server with the server with that id.

Any suggestions on how to update an object

**Comment by TheRealNate** [comment]
Hi @varunwyest, could you confirm that this works in your web client code? Your server must be configured with [allow deny](https://guide.meteor.com/security.html#allow-deny) for this to work.

**Comment by varunwyest** [comment]
Yes, I tested it thoroughly in the meteor app as well outside of react native the behaviour is exactly the same when we dont use the Object ID. Without the object id, just passing in the string it does not update document, but gives no error. 

When using the Object ID it updates the data perfectlty. I can try to take a stab at updating this code if you are okay with it. I just dont want to fork and create our own as this will be helpful to everybody.

**Comment by sdhimanwyest** [comment]
+1

**Comment by jankapunkt** [comment]
If this issue persist it's likely a minimongo issue. Feel free to reopen.
